first draft of intro (#87)
scale-vector/dlt: docs/website/docs/intro.md

# Introduction

![dlt pacman](/img/dlt-pacman.gif)

## Summary

> `dlt` is an open source library that you include in a Python script to create a highly 

Reviewer: ok this is good 🥇

> scalable, easy to maintain data [pipeline](./glossary.md#pipeline) that is straightforward 

Author: ```suggestion
scalable, easy to maintain data, straightforward to deploy [pipeline](./glossary.md#pipeline).
```

> scalable, easy to maintain data, straightforward to deploy [pipeline](./glossary.md#pipeline).

Author: ```suggestion
scalable, easy to maintain, straightforward to deploy data [pipeline](./glossary.md#pipeline).
```

> to deploy. Once you set it up, it will then automatically turn JSON returned by any 

Author: ```suggestion
Once you set it up, it will then automatically turn JSON returned by any 
```

[source](./glossary.md#source) (e.g. an API) into a live dataset stored in the 
[destination](./glossary.md#destination) of your choice (e.g. Google BigQuery).

## Who should use `dlt`?

Anyone who solves problems in their job using Python should use `dlt` (e.g. data scientists, 
data analysts, etc).

## What does `dlt` do?

`dlt` is used to automate fetching data for others, copying production data to somewhere else, putting data from an API into a database, getting data for dashboards that automatically refresh with new data, and more.

## How does `dlt` work?

> `dlt` extracts data from a [source](./glossary.md#source), normalizes it to a 

Author: ```suggestion
`dlt` extracts data from a [source](./glossary.md#source), inspects its structure to create a [schema](./glossary.md#schema), normalizes and verifies the data,
```

> `dlt` extracts data from a [source](./glossary.md#source), normalizes it to a 

Reviewer: `dlt` extracts data from a source, inspects its structure to create a [schema], normalizes and verifies the data, and then...

> [schema](./glossary.md#schema), and then loads the data into a [destination](./glossary.md#destination). 

Author: ```suggestion
and then loads the data into a [destination](./glossary.md#destination). 
```
